src/lpython/semantics/python_ast_to_asr.cpp

```diff
@@ -3638,13 +3637,18 @@ std::string pickle_tree_python(AST::ast_t &ast, bool colors) {
     return v.get_str();
 }
 
+std::string get_parent_dir(const std::string &path) {
+    int idx = path.size()-1;
+    while (idx >= 0 && path[idx] != '/' && path[idx] != '\\') idx--;
+    return path.substr(0,idx);
```

Comment:
```suggestion
    if (idx == -1) return "";
    return path.substr(0,idx);
```

Reply:
Let's send this as a separate PR, I need a quick fix in now, and we can do this improved one right after.

Reply:
This will raise an error if `path.substr(0, -1)`.

Reply:
Yes. Can you please submit a PR with a fix?